strategies/low_risk/.gitignore

```diff
@@ -0,0 +1,24 @@
+*node_modules
```

Comment:
what the point of adding .gitignore to this directory if we already have one in the root?

Reply:
fixed